test/e2e-test-application/src/main.ts

```diff
@@ -4,6 +4,9 @@ import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { AppModule } from './app/app.module';
 import { environment } from './environments/environment';
 
+import * as LuigiClient from '@luigi-project/client';
+window.LuigiClient = LuigiClient;
```

Comment:
the tag fix looks well. 
seems to fail on here : 
_`ERROR in src/main.ts(8,8): error TS2339: Property 'LuigiClient' does not exist on type 'Window'.`_

Reply:
I have fixed that one. Thanks.